modules/telestion-example/src/main/java/de/wuespace/telestion/example/SayHello.java

```diff
@@ -1,69 +1,42 @@
 package de.wuespace.telestion.example;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import io.vertx.core.AbstractVerticle;
+import de.wuespace.telestion.api.verticle.TelestionConfiguration;
+import de.wuespace.telestion.api.verticle.TelestionVerticle;
+import io.vertx.core.DeploymentOptions;
 import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+
 import java.time.Duration;
-import java.util.UUID;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import de.wuespace.telestion.api.config.Config;
 
 /**
  * A class which says hello and shows the usage of configuration files.
  *
  * @author Jan von Pichowski
```

Comment:
wanna update the author maybe? :D

Reply:
Maybe :wink:

Reply:
Done in [`169a520` (#430)](https://github.com/wuespace/telestion-core/pull/430/commits/169a520fa6285decb0e3409806941ed8daa0b2a0) :wink: